### ISSUE TITLE
Screen Reader: Many Pages: The informative error icon is not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76942]
## Action Performed:
1. Using JAWS + Chrome, open the page https://new.expensify.com/
2. Navigate using TAB past the phone or email input field without entering any information to generate the error message below it
3. Navigate to the error message using down arrow key navigation
4. Note if the error icon is announced.

Other Occurrences:
1. On Settings - Save the world - I know a teacher
2. On Settings - Security - Close account
3. On Settings - Security - Two-factor authentication
4. On Settings - Security - Two-factor authentication (step 2 of 2)
5. On Settings - Profile - Display name
6. On Settings - Profile - DOB
7. On Settings - Profile - Phone number
8. On Settings - Profile - Legal name
9. On Settings - Profile - Address
10. On Workspaces - Duplicate Workspaces
11. On Workspaces - Overview - Workspace Name
12. On Workspace - Categories - Add Category
13. On Workspace - Expensify Card - Bank info
14. On Workspace - Create Workspace - Confirm Workspace
15. On Create Report - Add payment card

## Expected Result:
Visually, the error message has an informative icon; the same should also be announced, e.g.:
- 'Error'

## Actual Result:
The informative error icon is not announced as part of the error message, nor announced separately.

HTML:
<div class=""css-g5y9jx r-18u37iz r-1awozwy r-knv0ih r-15zivkp""><div class=""css-g5y9jx"" data-testid=""""><svg xmlns=""[http://www.w3.org/2000/svg""](http://www.w3.org/2000/svg%22%22) xml:space=""preserve"" viewBox=""0 0 20 20"" width=""20"" height=""20"" hovered=""false"" pressed=""false"" fill=""#F25730""><circle cx=""10"" cy=""10"" r=""7.5""></circle></svg></div><div class=""css-g5y9jx r-13awgt0 r-1jkjb""><div dir=""auto"" class=""css-146c3p1"" style=""color: rgb(242, 87, 48); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; line-height: 16px; margin-bottom: 0px;"">Please enter an email or phone number</div></div></div>"

## Area issue was found in:
Landing Page

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Add an off-screen text ot announce the icon, e.g.:
- 'Error'
https://webaim.org/techniques/css/invisiblecontent/

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1916" height="1004" alt="Image" src="https://github.com/user-attachments/assets/4e88b623-b3f8-40be-b53f-b39ca474c48c" />

https://github.com/user-attachments/assets/8baec8ce-dc2a-4cd7-aa26-84a619196cad

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997285
**Issue reported by:** NA
**Slack conversation:** NA
